openwisp_firmware_upgrader/upgraders/openwrt.py

```diff
@@ -82,7 +82,7 @@ def _reflash(self, path):
         """
         def upgrade(conn, path, timeout):
             conn.connect()
-            conn.exec_command('sysupgrade -v -c {0}'.format(path),
+            conn.exec_command(f'sysupgrade -v -c {path}',
```

Comment:
do not use f-strings because we still want to support python 3.6 (f-strings are available from python 3.7 onwards)

Reply:
I think they are supported from 3.6 onwards.
 **Source:**[Python docs: What's new in 3.6](https://docs.python.org/3/whatsnew/3.6.html#whatsnew36-pep498)
I tested it locally on my python version 3.6.9 and it does work. Please correct me if I am wrong.

Reply:
@nemesisdesign, what about this? 😅

Reply:
You're right, from python 3.6 onwards :+1: